docusaurus/docs/Android/02-client/06-guides/03-channel-list-updates.mdx

```diff
@@ -0,0 +1,124 @@
+# Channel List Updates
+
+The `ChannelList` component is being updated based on the data from two sources:
```

Comment:
```suggestion
We update the `ChannelList` and `ChannelListView` components based on data from two sources:
```

Reply:
I think we want to refer to both compose and XML? Unless we can just say ChannelList

Reply:
I would add full stops `.` at the end of each bullet item here, as these seem like finished statements and examples.

I think we could do it for other bulleted lists too, unless we're just listing out items, e.g.

* ChannelList
* MessageList
* AttachmentsPicker

This would go w/o a full stop, as it's not a statement that can finish, but in these cases we're listing out and describing sth, so it might look like we have an unfinished sentence otherwise!

LMK what you think :D

Reply:
1. For some reason I thought we have the same names in both. Makes sense
2. Good idea!